powersoftau/src/utils.rs

```diff
@@ -213,7 +213,7 @@ fn dense_multiexp_inner<G: CurveAffine>(
     use std::sync::Mutex;
     // Perform this region of the multiexp. We use a different strategy - go over region in parallel,
     // then over another region, etc. No Arc required
-    let chunk = (bases.len() / num_cpus::get()) + 1;
+    let chunk = get_chunk_size(bases.len());
```

Comment:
Same as previous comment: This does not return the same value as the old implementation.

Reply:
This is intentional. The original implementation is not optimal.